resource_management/azure_mgmt_compute/spec/usage_operations_spec.rb

```diff
@@ -8,8 +8,9 @@
 include Azure::ARM::Compute
 
 describe UsageOperations do
-  before(:all) do
-    @client = COMPUTE_CLIENT.usage_operations
+  before(:each) do
+    @resource_helper = ResourceHelper.new
```

Comment:
Yup that can be handled way better. updated it to use lazy initialization of class attributes.